inst/isotopes/isotope_definition.txt

```diff
@@ -0,0 +1,289 @@
+"element"	"isotope"	"exact.mass"	"rel.abundance"
```

Comment:
Could you please rename the columns to `"exact_mass"` and `"rel_abundances"` - this is a ~ somewhat convention we have for column names (i.e. use `"_"` as separator).